.gitlab-ci.yml

```diff
@@ -29,7 +29,7 @@ before_script:
         - docker info
         - sleep 300
         - echo --------- INNER CONTAINER -------------
-        - docker run --rm -i -e CVMFS_MOUNTS="none" -v ${PWD}/:/home/cmsuser/workdir/ ${BASE_IMAGE} -- -i -c "tar -czf ${CMSSW_VERSION}.tar.gz --exclude .git* ${CMSSW_VERSION} && mv ${CMSSW_VERSION}.tar.gz /home/cmsuser/workdir/  && cd /home/cmsuser/workdir/ && pwd && ls -alh ./ && cd /home/cmsuser/"
+        - docker run --rm -i -e CVMFS_MOUNTS="none" -v ${PWD}/:/home/cmsusr/workdir/ ${BASE_IMAGE} -- -i -c "tar -czf ${CMSSW_VERSION}.tar.gz --exclude .git* ${CMSSW_VERSION} && mv ${CMSSW_VERSION}.tar.gz /home/cmsusr/workdir/  && cd /home/cmsusr/workdir/ && pwd && ls -alh ./ && cd /home/cmsusr/"
```

Comment:
would `$HOME` work here?

Reply:
Partially. As you can see from my changes, I've used home in all of the places where we're inside the container. However, I can't use the `HOME` environment variable when setting up the bind mount because that's outside of the cms-cvmfs-docker container and the `HOME` variable in the `docker:latest` image is different.